### ISSUE TITLE
no-commonjs-return flags all return statements inside of a module definition

### ISSUE THREAD

**Issue by cvisco** [cvisco/eslint-plugin-requirejs#36]
The following trivial example results in an error, and should not:

``` js
define(function (require, exports, module) {

    function addOne(thing) {
        return thing + 1; // flagged as error
    }

    module.exports = {
        addOne: addOne
    };
});
```
